net-misc/ssh-askpass-fullscreen/files/ssh-askpass-fullscreen.1

```diff
@@ -1,79 +0,0 @@
-.\"                                      Hey, EMACS: -*- nroff -*-
-.\" First parameter, NAME, should be all caps
-.\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
-.\" other parameters are allowed: see man(7), man(1)
-.TH SSH-ASKPASS-FULLSCREEN 1 "May 8 , 2004"
-.\" Please adjust this date whenever revising the manpage.
-.\"
-.\" Some roff macros, for reference:
-.\" .nh        disable hyphenation
-.\" .hy        enable hyphenation
-.\" .ad l      left justify
-.\" .ad b      justify to both left and right margins
-.\" .nf        disable filling
-.\" .fi        enable filling
-.\" .br        insert line break
-.\" .sp <n>    insert n+1 empty lines
-.\" for manpage-specific macros, see man(7)
-.SH NAME
-ssh-askpass-fullscreen \- A simple replacement for ssh-askpass written with gtk2
-.SH SYNOPSIS
```

Comment:
Apparently I need to use start a review instead of "add single comment" if I have only one note to review, to actually get it show up as a "changes requested"...